analytics.proto

```diff
@@ -0,0 +1,54 @@
+syntax = "proto3";
+
+package livekit;
+option go_package = "github.com/livekit/protocol/proto/livekit";
+
+import "google/protobuf/empty.proto";
+import "google/protobuf/timestamp.proto";
+
+service AnalyticsRecorderService {
+  rpc IngestStats(stream  AnalyticsStats) returns (google.protobuf.Empty){};
+  rpc IngestEvents(stream AnalyticsEvents) returns (google.protobuf.Empty){};
+}
+
+enum STATSSTREAMTYPE {
+  STATSUPSTREAM = 0;
+  STATSDOWNSTREAM = 1;
+}
+
+message AnalyticsStat {
+  STATSSTREAMTYPE kind = 1;
+  google.protobuf.Timestamp timestamp = 2;
+  string node = 3;
+  optional string SID = 4;
+  optional string projectId = 5;
+  optional string participantId = 6;
+  optional string roomName = 7;
+  optional double jitter = 8;
+  optional uint64 packetLost = 9;
+  optional uint64 rrtime = 10;
+  optional uint64 bytesSent = 11;
+  optional uint64 bytesReceived = 12;
+  optional string authToken = 13;
+}
+
+message AnalyticsStats {
+  repeated AnalyticsStat stats = 1;
+}
+
+enum EventSignalTarget {
```

Comment:
It feels that we might not need this. most events are agnostic of the direction, and for ones that are, we should re-use `StreamType`